xbmc/cores/VideoPlayer/MediaPipelineWebOS.h

```diff
@@ -37,9 +37,9 @@ class CDVDAudioCodec;
 class StarfishMediaAPIs;
 
 /**
-  * @class CMediaPipelineWebOS
-  * @brief WebOS media pipeline for audio/video playback.
-  */
+* @class CMediaPipelineWebOS
```

Comment:
Your IDE just does not like you.

Reply:
Looks alright to me? I don't know what the exact formatting rules on doxygen comments are. I looked at a couple of files for reference but it's all over the place and clang-format generally doesn't format these at all

Reply:
I just found `ReflowComments: IndentOnly` which does allow clang-format to properly work. Should be alright now. Maybe we should set that clang-format option by default though.